lib/client/websocket-tracker.js

```diff
@@ -215,6 +242,32 @@ WebSocketTracker.prototype._onSocketData = function (data) {
   }
 }
 
+WebSocketTracker.prototype._onScrapeResponse = function (data) {
+  var self = this
+  // NOTE: the unofficial spec says to use the 'files' key, 'host' has been
+  // seen in practice
+  data = data.files || data.host || {}
```

Comment:
We should just use `files` since this is the first implementation. We don't need to support hacks for bad tracker servers.